src/py/migraphx_py.cpp

```diff
@@ -20,6 +22,94 @@
 using half   = half_float::half;
 namespace py = pybind11;
 
+namespace migraphx {
+migraphx::value kwargs_to_value(py::kwargs kwargs);
+
+migraphx::value list_to_value(py::list lst)
+{
+    migraphx::value v = migraphx::value::array{};
+    for(auto&& val : lst)
+    {
+        if(py::isinstance<py::kwargs>(val))
+        {
+            auto elem_v = kwargs_to_value(val.cast<py::kwargs>());
+            v.push_back(elem_v);
+        }
+        else if(py::isinstance<py::list>(val))
+        {
+            auto elem_v = list_to_value(val.cast<py::list>());
+            v.push_back(elem_v);
+        }
+        else if(py::isinstance<py::bool_>(val))
+        {
+            v.push_back(val.cast<bool>());
+        }
+        else if(py::isinstance<py::int_>(val))
+        {
+            v.push_back(val.cast<int>());
+        }
+        else if(py::isinstance<py::float_>(val))
+        {
+            v.push_back(val.cast<float>());
+        }
+        else if(py::isinstance<py::str>(val))
+        {
+            v.push_back(val.cast<std::string>());
+        }
+        else
+        {
+            MIGRAPHX_THROW("LIST_TO_VALUE: Unsupported data type ");
+        }
+    }
+
+    return v;
+}
+
+migraphx::value kwargs_to_value(py::kwargs kwargs)
```

Comment:
Should be named `migraphx_to_value` so it will be called automatically by `migraphx::to_value`.